File: ServerTemplate.js/restserver.js
var restify = require("restify");

var server = restify.createServer();

server.get("/nouns", function (request, response, next) {
    response.end("All nouns details are listed");
    return next();
});

server.get("/nouns/:id", function (request, response, next) {
    var id = request.params.id;
    response.end("noun with id " + id + " is listed");
    return next();
});

server.post("/nouns/:id", function (request, response, next) {
    var id = request.params.id;
    response.end("noun details of " +  id  + " is inserted successfully");
    return next();
});

server.put("/nouns/:id", function (request, response, next) {
    var id = request.params.id;
    response.end("noun with id " + id + " is updated");
    return next();
});

server.del("/nouns/:id", function (request, response, next) {
    var id = request.params.id;
    response.end("noun with id " + id + " is deleted");
    return next();
});

server.listen(5000, "0.0.0.0", function() {
    console.log("Listening to the address " + server.url)
});

File: ServerTemplate.py/src/restserver.py
from flask import Flask
from flask import jsonify
from flask import make_response

app = Flask(__name__)

@app.route('/nouns', methods=['GET'])
def get_nouns():
    return 'All nouns details are listed'
    
@app.route('/nouns/<id>', methods=['GET'])
def get_noun(id):
    return 'the noun with ' + id + ' is listed'
	
@app.route('/nouns/<id>', methods=['POST'])
def create_noun(id):
	return 'the noun with ' + id + ' is inserted successfully'	

@app.route('/nouns/<id>', methods=['PUT'])
def update_noun(id):
	return 'the noun with ' + id + ' is updated'
	
@app.route('/nouns/<id>', methods=['DELETE'])
def delete_noun(id):
	return 'the noun with ' + id + ' is deleted'

@app.errorhandler(404)
def not_found(error):
    return make_response(jsonify({'error': 'Not found'}), 404)
    

if __name__ == '__main__':
    app.run(debug = True, host = '0.0.0.0')
